src/group-configurations/experiment-configurations-section/ExperimentFormGroups.jsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/no-array-index-key */
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Close as CloseIcon, Add as AddIcon } from '@edx/paragon/icons';
+import {
+  Form, Icon, IconButtonWithTooltip, Stack, Button,
+} from '@edx/paragon';
+
+import {
+  getNextGroupName,
+  getGroupPercentage,
+  getFormGroupErrors,
+} from './utils';
+import messages from './messages';
+
+const ExperimentFormGroups = ({
+  groups,
+  errors,
+  onChange,
+  onDeleteGroup,
+  onCreateGroup,
+}) => {
+  const { formatMessage } = useIntl();
+  const percentage = getGroupPercentage(groups.length);
+  const { arrayErrors, stringError } = getFormGroupErrors(errors);
+
+  return (
+    <Form.Group className="form-group-configuration">
+      <Form.Label className="font-weight-bold">
+        {formatMessage(messages.experimentConfigurationGroups)}*
+      </Form.Label>
+      <Form.Control.Feedback className="mt-0">
+        {formatMessage(messages.experimentConfigurationGroupsFeedback)}
+      </Form.Control.Feedback>
+      {stringError && (
+        <Form.Control.Feedback type="invalid" hasIcon={false}>
+          {stringError}
+        </Form.Control.Feedback>
+      )}
+      <Stack className="mt-3">
+        {groups.map((group, idx) => {
+          const fieldError = arrayErrors?.[idx]?.name;
+          const isInvalid = !!fieldError;
+
+          return (
+            <Form.Group
+              key={idx}
```

Comment:
```suggestion
              key={group.name}
```
will it be unique or maybe `group.id`? If you change here you can remove `eslint-disable` on top of the file

Reply:
```
export const initialExperimentConfiguration = {
  name: '',
  description: '',
  groups: [
    { name: 'Group A', version: 1, usage: [] },
    { name: 'Group B', version: 1, usage: [] },
  ],
  scheme: 'random',
  parameters: {},
  usage: [],
  active: true,
  version: 1,
};
```

This component handle each group item, so we don't always have `id` field

Reply:
Because it is initial groups without preinstalled `id` value